client/my-sites/plans-features-main/components/plan-notice.tsx

```diff
@@ -0,0 +1,101 @@
+import { PlanSlug } from '@automattic/calypso-products';
+import { formatCurrency } from '@automattic/format-currency';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+import MarketingMessage from 'calypso/components/marketing-message';
+import Notice from 'calypso/components/notice';
+import { getDiscountByName } from 'calypso/lib/discounts';
+import { usePlanUpgradeCreditsDisplay } from 'calypso/my-sites/plans-features-main/hooks/use-plan-upgrade-credits-display';
+import { getCurrentUserCurrencyCode } from 'calypso/state/currency-code/selectors';
+import { isCurrentUserCurrentPlanOwner } from 'calypso/state/sites/plans/selectors';
+import { isCurrentPlanPaid } from 'calypso/state/sites/selectors';
+
+export default function PlanNotice( {
+	siteId,
+	isInSignup,
+	visiblePlans = [],
+	discountInformation: { withDiscount, discountEndDate },
+}: {
+	visiblePlans: PlanSlug[];
+	isInSignup: boolean;
+	siteId: number;
+	discountInformation: {
+		withDiscount: string;
+		discountEndDate: Date;
+	};
+} ) {
+	const translate = useTranslate();
+	const [ isNoticeDismissed, setIsNoticeDismissed ] = useState( false );
+	const currencyCode = useSelector( ( state ) => getCurrentUserCurrencyCode( state ) );
+	const canUserPurchasePlan = useSelector(
+		( state ) =>
+			! isCurrentPlanPaid( state, siteId ) || isCurrentUserCurrentPlanOwner( state, siteId )
+	);
+	const activeDiscount = getDiscountByName( withDiscount, discountEndDate );
+	const { creditsValue, isPlanUpgradeCreditEligible } = usePlanUpgradeCreditsDisplay(
+		siteId,
+		visiblePlans
+	);
+
+	const handleDismissNotice = () => setIsNoticeDismissed( true );
+
+	if ( isNoticeDismissed ) {
+		return null;
+	} else if ( ! canUserPurchasePlan ) {
+		return (
+			<Notice
+				className="plan-features__notice"
+				showDismiss={ true }
+				onDismissClick={ handleDismissNotice }
+				status="is-info"
+			>
+				{ translate(
+					'This plan was purchased by a different WordPress.com account. To manage this plan, log in to that account or contact the account owner.'
+				) }
+			</Notice>
+		);
+	} else if ( activeDiscount ) {
+		return (
+			<Notice
+				className="plan-features__notice-credits"
```

Comment:
@jdc91 just FYI - the classes passed here don't have significance it seems e.g. there's no styling depended on them. 

We will still want to pass something for updating some of the inner styles/margins added. This is something I'm working out on my end (for the reskin https://github.com/Automattic/wp-calypso/pull/75333). I am currently passing `plan-features-main__notice` for all of them.

Reply:
I think this is a change for the <Notice />  component right? Are we not changing this styling system wide?

Reply:
Yes, we are (changing it system wide). It's just for a case of a bottom margin that I chose to only remove for these notices. sounds like something I may need investigate better though. I guess a Notice shouldn't really include a margin by default to begin with 🤔